ddtrace/internal/runtime/runtime_metrics.py

```diff
@@ -60,31 +60,20 @@ class RuntimeWorker(_worker.PeriodicWorkerThread):
 
     FLUSH_INTERVAL = 10
 
-    def __init__(self, statsd_client, flush_interval=None):
-        flush_interval = self.FLUSH_INTERVAL if flush_interval is None else flush_interval
+    def __init__(self, statsd_client, flush_interval=FLUSH_INTERVAL):
         super(RuntimeWorker, self).__init__(interval=flush_interval,
                                             name=self.__class__.__name__)
         self._statsd_client = statsd_client
         self._runtime_metrics = RuntimeMetrics()
 
-    def _write_metric(self, key, value):
-        log.debug('Writing metric {}:{}'.format(key, value))
-        self._statsd_client.gauge(key, value)
-
     def flush(self):
-        if not self._statsd_client:
```

Comment:
was this just unnecessary?

Reply:
I hope not. I mean we're not going to check every 10 seconds if we've been init correctly, right? 😓 
The `Tracer` always pass a stats client, fortunately.

Reply:
👍